src/command_background.ts

```diff
@@ -0,0 +1,323 @@
+import { List, Map } from "immutable"
+import { MayList, toList, HistItem, RArray } from "./util"
+import minimist from "./minimist"
```

Comment:
I can't remember why, but in #523 we found that minimist and yargs didn't do quite what we wanted, hence why we went with command-line-args, which still doesn't quite do what we wanted, but seemed less insurmountably so at the time.